src/entry-server.js

```diff
@@ -8,6 +8,7 @@ export default function initializeServer(createApp, serverOpts) {
         i18nLoader: null,
         vuexModules: true,
         logger: console,
+        additionalContext: null,
```

Comment:
Moved and renamed based on your feedback. Now it takes anything passed in to additional context and will merge it in to the context object below via `Object.assign`.